.github/workflows/main.yml

```diff
@@ -493,6 +493,24 @@ jobs:
         name: tarballs
         path: dist
 
+    # The action 'pypa/gh-action-pypi-publish' will try to upload all files in the
+    # dist/ folder. This folder also contains non-package files, and therefore the
+    # action fails.
+    #
+    # To prevent the action from failing all .whl files are copied into a new
+    # directory.
+    - run: |
+        mkdir -p tmp/whl
+        find dist/ -name '*.whl' -type f -exec cp '{}' tmp/whl -v \;
+
+    - name: Publish Python wheels on Pypi
+      uses: pypa/gh-action-pypi-publish@master
```

Comment:
Is there perhaps a tag or a revision we could use instead of relying on `master`? 

(that way if this changes upstream we don't break and we have time to update ourselves)

Reply:
Sure, version 1.0.0a0 is the latest (and only) version. I'll change it.